osquery/tables/system/posix/augeas.cpp

```diff
@@ -15,13 +15,25 @@
 #include <osquery/logger.h>
 #include <osquery/tables.h>
 
+namespace osquery {
+
+/**
+ * @brief Augeas lenses path.
+ *
+ * Directory that contains augeus lenses.
+ */
 #ifdef __APPLE__
-#define LENSES_PATH "/private/var/osquery/lenses"
+FLAG(string,
```

Comment:
Can you bring the `FLAG` macro out of the `ifdef`, and just have 1 flag definition with the variable `LENSES_PATH` macro?